CHANGELOG.md

```diff
@@ -40,6 +40,17 @@ All notable changes to this project will be documented in this file.
 ## Unreleased
 * ### ALL
     * #### Added
+        * Added docs_static/nirfsg_*.inc files for nirfsg 
+    * #### Changed
+        * Updated conf.py.mako to include RTD link for nirfsg.
+        * Updated about.inc, installation.inc, documentation.inc, README.rst, makefile, tox.ini and tox-travis.ini files to have nirsg generated.
+    * #### Removed
+* ### `nirfsg` (NI-RFSG)
+    * #### Added
+        * Enabled nirfsg module support and added metadata for selected public APIs
```

Comment:
since this is markdown, let's try to use backticks so that symbols are displayed as such. E.g. paths or the `nirfsg` module name.

I understand we haven't been perfect about this, and it is something we culd fix in its own PR. But at least for additions...